QUICKSTART.md

```diff
@@ -67,9 +77,21 @@ kubectl ws tree
 
 ## 2. Example deployment of Apache HTTP Server workload into two local kind clusters
 
+In this example you will create two edge clusters and define one
+workload that will be distributed from the center to those edge
```

Comment:
I do appreciate these extra words. I'm somewhat confused as to why they are already on the docs site? I tried to see the 'before/after' and these words already appear at https://docs.kubestellar.io/release-v0.2.2/Getting-Started/quickstart/

So, I'm kinda confused by that. Almost like this PR should target v0.2.2 not main?

Reply:
I created this PR before some other changes were made to the website.  Some of those changes also added this particular words.

Reply:
Also, the `v0.2.2` tag has been set, as far as I know we can not change it (or even if we could, we should not change it, because tags are expected to be immutable).  Right now we are not maintaining release branches a la k8s.io/kubernetes; we just develop on `main` and occasionally tag a release.